employee/models.py

```diff
@@ -66,12 +84,24 @@ def picture_tag(self):
 
     # Save Employee Data
     def save(self,  *args, **kwargs):
-        # Insert data
-        if self.id:
-            self.updatedBy = get_current_authenticated_user()
-            self.updatedAt = datetime.datetime.now()
         # Update data
+        if self.id:
+            self.updated_by = get_current_authenticated_user()
+        # Insert data
         else:
-            self.createdBy = get_current_authenticated_user()
-            self.updatedBy = get_current_authenticated_user()
+            self.created_by = get_current_authenticated_user()
+            self.updated_by = get_current_authenticated_user()
         super(Employee, self).save(*args, **kwargs)
+
+    def has_module_perms(self, app_label):
+        if self.is_superuser is True:
+            return True
+        elif self.is_staff is True and self.is_superuser is False:
+            if app_label is 'department':
+                return False
+        return True
```

Comment:
`is_superuser`/`is_staff` is a boolean field, so we don't have to check for True or False like this.
Also, `is` check sif the objects are the same so let's use `==` if we are just checking values.

```suggestion
    def has_module_perms(self, app_label):
        if not self.is_superuser and self.is_staff and app_label == 'department':
            return False
        return True
```

Still, I am confused about the logic here. What are we trying to do here?

Reply:
Okay, so we basically want this?

```python
        def has_module_perms(self, app_label):
            return self.is_superuser or app_label != 'department':
```

Reply:
Thanks.